docs/src/examples/misc_fr.adoc

```diff
@@ -1,13 +0,0 @@
-= Entrée de palpeur de mesure
-
-Pour utiliser une sonde de mesure avec G38 un palpeur doit être
-raccordé à *motion.probe-input*. Dans cet exemple, la broche 15 du port
-parallèle est utilisée. Dans votre fichier .hal ajoutez la ligne
-suivante. Elle doit bien sur être adaptée à l'entrée sur laquelle votre
-palpeur est raccordé.
-----
-net probe-in motion.probe-input <= parport.0.pin-15-in
-----
-
-Pour d'autres informations concernant la sonde de mesure voir la documentation
-du G-code, les chapitres relatifs à G38.
```

Comment:
Would have better moved it to archive/fr/ subtree to not burry it in git history before french translation is done.

Reply:
I agree. I may just have been overwhelmed with the differences.